clouddriver-oracle-bmcs/src/main/groovy/com/netflix/spinnaker/clouddriver/oraclebmcs/provider/agent/OracleBMCSNetworkCachingAgent.groovy

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2017 Oracle America, Inc.
+ *
+ * The contents of this file are subject to the Apache License Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * If a copy of the Apache License Version 2.0 was not distributed with this file,
+ * You can obtain one at https://www.apache.org/licenses/LICENSE-2.0.html
+ */
+package com.netflix.spinnaker.clouddriver.oraclebmcs.provider.agent
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.netflix.spinnaker.cats.agent.AgentDataType
+import com.netflix.spinnaker.cats.agent.CacheResult
+import com.netflix.spinnaker.cats.agent.DefaultCacheResult
+import com.netflix.spinnaker.cats.cache.CacheData
+import com.netflix.spinnaker.cats.cache.DefaultCacheData
+import com.netflix.spinnaker.cats.provider.ProviderCache
+import com.netflix.spinnaker.clouddriver.oraclebmcs.cache.Keys
+import com.netflix.spinnaker.clouddriver.oraclebmcs.security.OracleBMCSNamedAccountCredentials
+import com.oracle.bmc.core.model.Vcn
+import com.oracle.bmc.core.requests.ListVcnsRequest
+import groovy.util.logging.Slf4j
+
+@Slf4j
+class OracleBMCSNetworkCachingAgent extends AbstractOracleBMCSCachingAgent {
+
+  final Set<AgentDataType> providedDataTypes = [
+    AgentDataType.Authority.AUTHORITATIVE.forType(Keys.Namespace.NETWORKS.ns)
+  ] as Set
+
+  OracleBMCSNetworkCachingAgent(String clouddriverUserAgentApplicationName,
+                                OracleBMCSNamedAccountCredentials credentials,
+                                ObjectMapper objectMapper) {
+    super(objectMapper, credentials, clouddriverUserAgentApplicationName)
+  }
+
+  @Override
+  CacheResult loadData(ProviderCache providerCache) {
+    List<Vcn> vcns = loadVcns()
+    buildCacheResult(vcns)
```

Comment:
Please add `return` statements to the methods that aren't just one-liners. (It's not a formal style requirement, but we're trying to be more consistent about it.)